curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/comparisons-with-the-logical-or-operator.md

```diff
@@ -25,7 +25,7 @@ if (num < 5) {
 return "Yes";
 ```
 
-will return `Yes` only if `num` is between `5` and `10` (5 and 10 included). The same logic can be written as:
+This will return `Yes` only if `num` is between `5` and `10` (`5` and `10` included).  The same logic can be written with the <dfn>logical or</dfn> operator.
```

Comment:
minor consistency suggestion
```suggestion
This code will return `Yes` if `num` is between `5` and `10` (`5` and `10` included). The same logic can be written with the <dfn>logical or</dfn> operator.
```

Reply:
Thank you and it has now been completed.